<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestGestionStock</class>
 <widget class="QWidget" name="TestGestionStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestGestionStock</string>
  </property>
  <widget class="QPushButton" name="pushButtonNouveau">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRetrait">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retrait</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSuppression">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Suppression</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStock">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rouleaux en stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRetenus">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Rouleaux retenus</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetStock">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>161</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetRouleaux">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>161</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelDiametre">
      <property name="text">
       <string>Diamètre :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDiametre"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelReference">
      <property name="text">
       <string>Référence :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditReference"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>TestGestionStock</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
